### ISSUE TITLE
Is there a list of blocked urls, and reason of blocking?

### ISSUE THREAD

**Issue by mayli** [andrieslouw/imagesweserv#285]
```
{
"status": "error",
"code": 404,
"message": "The hostname of the origin is unresolvable (DNS) or blocked by policy."
}
```
This seems to be a bug

"GET /?url=http://up.ccp.ovh/files/NrjVd/e6.png&w=240&il&output=webp HTTP/1.1" 200 5892 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.85 Safari/537.36" 1619573101.909
"GET /?url=http://up.ccp.ovh/files/NrjVd/e6.png&w=240&il&output=webp HTTP/1.1" 404 132 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.85 Safari/537.36" 1619573102.982

The first requests went through, but the second or consecutive ones got 404.

**Comment by andrieslouw** [comment]
Normally all urls are parsed by https://community.opendns.com/domaintagging/

To prevent our public service from being blocked or attacked by DMCA's, we block these categories:
- Adware
- Anime/Manga/Webcomic
- Gambling
- Hate/discrimination
- Nudity
- Pornography
- Proxy's (to mitigate the risk of redirect-loops)
- Sexuality
- Tasteless

Due to recent abuse and DMCA complaints, we added a list of domains ourselves on top of this. I will need to check if your domain is in this list.

Please use the source code provided here to host your own solution if you're blocked.

**Comment by andrieslouw** [comment]
Just checked: ccp.ovh was blocked by script, due to related DMCA issues. I removed the block for now, and will monitor again.

**Comment by espressoelf** [comment]
Sorry for hijacking this, but answering here seems to be more appropriate than creating a new issue. Is **live.flickr.com** on your blocklist, too? OpenDNS didn't change their assigned tags as far as I can see, but I get the same error as mentioned in the initial post.

**Comment by andrieslouw** [comment]
`flickr.com` is not on our list (and also no subdomains), but `live.flickr.com` gives `NXDOMAIN`. I asked the nameservers of `flickr.com`, but they cannot find `live.flickr.com`. Are you sure that `live.flickr.com` exists?

**Comment by espressoelf** [comment]
I'm sorry, looks like I typo'd, should've copy&pasted it. **live.staticflickr.com** is the one I'm talking about. It inherits the CDN tag from Cloudfront on OpenDNS.

**Comment by andrieslouw** [comment]
Thank you, removed it from our list, it triggered our abuse detection scripts. Should work again soon(ish)!
